src/constants/application.ts

```diff
@@ -0,0 +1,5 @@
+const APPLICATION = {
+  NAME: 'tfs',
```

Comment:
Is this related to `name` in `src/config/app.config.ts` in any meaningful way?

Reply:
No, since I am awaiting your PR to be merged into `main` first I have created a dummy constant file for now. However I will raise an another PR where all abstractable constants from `app.config.ts` will be referred from `@ukef/constants`.

Reply:
That makes sense, thanks